### ISSUE TITLE
Appreciate 

### ISSUE THREAD

**Issue by hdt25** [krreet/plugin.video.jiotv#36]
Not an issue. But writing here to just appreciate your work. You brought a long awaited feature (Streaming Quality) 
Which I was looking for past 3 years.
Very much thanks for that feature.
It makes streaming more easy than before on kodi , we can lower the quality to be free from buffering video and continuous playback ♥️

**Comment by sun1423** [comment]
> Not an issue. But writing here to just appreciate your work. You brought a long awaited feature (Streaming Quality) Which I was looking for past 3 years. Very much thanks for that feature. It makes streaming more easy than before on kodi , we can lower the quality to be free from buffering video and continuous playback ♥️

yes this is must hv issue, in case buffering , this issue can be controlled by low video output if something not entirely depends from source. buffering and hanging are two negative.. otherwise gr8 work 🙏✨

**Comment by Kedharnadh** [comment]
Thank a lot for the support on the plugin and epg url option is a great feature to add custom epg
